include/quda.h

```diff
@@ -1524,37 +1524,47 @@ extern "C" {
   /**
    * Performs APE smearing on gaugePrecise and stores it in gaugeSmeared
    * @param n_steps Number of steps to apply.
-   * @param alpha  Alpha coefficient for APE smearing.
+   * @param alpha   Alpha coefficient for APE smearing.
    * @param meas_interval Measure the Q charge every Nth step
    */
-  void performAPEnStep(unsigned int n_steps, double alpha, int meas_interval);
+  void performAPEnStep(unsigned int n_steps, double alpha, int meas_interval,
+		       QudaGaugeObservableParam *param);
```

Comment:
While we're updating these interface functions, I wonder if we should take the opportunity to fix their names.  QUDA interface functions should end in Quda, e.g., `performanceAPEnStepQuda`.

Reply:
Also, do any of these parameters (`alpha`, `n_steps`, `meas_interval`) belong in the `QudaGaugeObervableParam` struct)?

Reply:
> While we're updating these interface functions, I wonder if we should take the opportunity to fix their names. QUDA interface functions should end in Quda, e.g., `performanceAPEnStepQuda`.

That's easy fix and certainly should be applied.

> Also, do any of these parameters (`alpha`, `n_steps`, `meas_interval`) belong in the `QudaGaugeObervableParam` struct)?

Not at the moment, but we could add generic "meas_n_steps", "alpha", "rho", "mean_interval" which could serve for all three smearing types and flow types with just a bit of documentation.